grpc_streaming_attestation/src/server.rs

```diff
@@ -114,7 +114,7 @@ where
         Ok(Self {
             tee_certificate,
             request_handler,
-            additional_info,
+            additional_info: Arc::new(additional_info),
```

Comment:
Do we prefer a let declaration over instantiating the Arc in the struct instantiation?